hal/lib/athena/handles/IndexedHandleResource.h

```diff
@@ -33,12 +36,13 @@ namespace hal {
 template <typename THandle, typename TStruct, int16_t size,
           HalHandleEnum enumValue>
 class IndexedHandleResource {
+ friend class IndexedHandleResourceTest;
```

Comment:
I had thought I had ran format.py, but must have forgot. It moved a few other things too.